src/Proto.Cluster/ClusterContracts.proto

```diff
@@ -43,6 +43,7 @@ message ProxyActivationRequest {
 
 message ActivationResponse {
     actor.PID pid = 1;
+    bool failed = 2;

```

Comment:
add Failed prop to ActivationResponse. instead of relying on null pid which is a bit semantically undefined.